keyboards/oddball/readme.md

```diff
@@ -3,11 +3,18 @@
 A split keyboard with trackball support.
 
 * Keyboard Maintainer: [Alexander Tulloh](https://github.com/atulloh)
-* Hardware Supported: Oddball PCB and plates, and using an [ADNS9800 and breakout board for mouse](https://www.tindie.com/products/jkicklighter/adns-9800-laser-motion-sensor/)
+* Hardware Supported: Oddball PCB and plates, and using an [ADNS9800](https://www.tindie.com/products/jkicklighter/adns-9800-laser-motion-sensor/) or a [PMW3360](https://www.tindie.com/products/jkicklighter/pmw3360-motion-sensor/) breakout board for mouse
 * Hardware Availability: [atulloh.github.io/oddball](https://atulloh.github.io/oddball)
 
 Make example for this keyboard (after setting up your build environment):
 
     make oddball:default
+    
```

Comment:
We shouldn't suggest changes at the keyboard level, and instead the suggestion should be for how to change it at the keymap level. Another solution is to set up the directory structure to have the sensor type as a sub-revision.
```
oddball/v1/
oddball/v2/adns_9800/
oddball/v2/pwm_33600/
```

Reply:
Ah, ok, that clarifies things a bit. Would leaving the existing keymap and just adding a new one for the new PMW sensor be acceptable?

```
oddball/keymaps/default (no change, still uses ADNS)
oddball/keymaps/pmw3360
```

Reply:
I did something along these lines. The advantage is that you don't need to add multiple keyboard sub-revisions every time a version gets added.